services/visual-studio-app-center/visual-studio-app-center-builds.tester.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('[fixed] Passing Build')
```

Comment:
I think you could probably collapse these first four tests into a single test, and just use the `isBuildStatus` validator for the message vs. checking for an explicit message with fixed status builds/mocked responses. 

The core focus for the service tests is to validate integration with the upstream API and to try to catch integration/contract issues quickly, and we can cover that with a single test. The rest are basically just re-testing the logic of the `renderBuildStatusBadge` function which is heavily tested already elsewhere